logbook-api/src/main/java/org/zalando/logbook/internal/JsonMediaType.java

```diff
@@ -1,4 +1,4 @@
-package org.zalando.logbook.json;
+package org.zalando.logbook.internal;
```

Comment:
`org.zalando.logbook.internal` package is already defined in the `logbook-api` module. I wish I could move everything in that package to `org.zalando.logbook.api`, but it would have been a very severe breaking change.

In order to be Java 9 module compatible, let's keep packages separate. Here I suggest using the following package instead:
```suggestion
package org.zalando.logbook.json.internal;
```

Reply:
The problem here is that I actually moved the file to the api module so that I can use it in `org.zalando.logbook.ContentType` class. That's one of the changes I'm concerned about, but as we use default methods in the api interfaces, it seems to be a necessary evil. If you have good ideas how to avoid this without major breaking changes, please share.